src/components/date/navbar/__snapshots__/__spec__.js.snap

```diff
@@ -0,0 +1,9 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Navbar Navbar Button renders presentational div and context provider for its children 1`] = `
+<button
+  className="sc-bdVaJa fCIlHW"
+>
+  sample children
+</button>
+`;
```

Comment:
you should import `jest styled-components`

Reply:
added, thanks